cfgov/jinja2/v1/_includes/organisms/bureau-structure.html

```diff
@@ -0,0 +1,334 @@
+         and titles (list of titles the person holds in particular office).
+
+   ========================================================================== #}
+{% macro _role(role) -%}
+<div class="o-bureau-structure_role">
+    {% if role.office_title %}
+    <h6 class="o-bureau-structure_role_office">
+        {{ role.office_title | safe }}
+    </h6>
+    {% endif %}
+    {% if role.name %}
+    <span class="o-bureau-structure_role_name">
+        {{ role.name | safe }}
+    </span>
+     {% endif %}
+    {% for title_line in ['line_1', 'line_2'] if role.title %}
+    <span class="o-bureau-structure_role_title">
+        {{  role.title[title_line] | safe }}
+    </span>
+    {% endfor %}
+</div>
+{%- endmacro %}
+
+
+{# ==========================================================================
+
+   _legend()
+
+   ==========================================================================
+
+   Description:
+
+   Renders an org chart legend.
+
+   ========================================================================== #}
+{% macro _legend() -%}
+<section class="o-bureau-structure_legend">
+    <h4>Chart Legend</h4>
+    <ul class="list
+               list__unstyled
+               list__spaced">
+        <li class="list_item">
+            <span class="o-bureau-structure_legend_symbol">*</span>
+            <span class="o-bureau-structure_legend_definition">
+                Position currently filled on an acting basis
+            </span>
+        </li>
+        <li class="list_item">
+            <span class="o-bureau-structure_legend_symbol">**</span>
+            <span class="o-bureau-structure_legend_definition">
+                Position has direct reporting responsibilities to the director
+            </span>
+        </li>
+        <li class="list_item">
+            <span class="o-bureau-structure_legend_symbol">***</span>
+            <span class="o-bureau-structure_legend_definition">
+                Position is not part of the CFPB director’s office
+            </span>
+        </li>
+</section>
+{%- endmacro %}
+
+
+{# ==========================================================================
+
+   _footer()
+
+   ==========================================================================
+
+   Description:
+
+   Renders an org chart footer:
+
+   ========================================================================== #}
+{% macro _footer() -%}
+<aside class="block
+              block__padded-top
+              block__border-top
+              o-bureau-structure_footer">
+    <div class="content-l content-l__main">
+        <div class="content-l_col content-l_col-1-2"
+             data-qa-hook="org-chart-download-info">
+            <h2 class="h3">Download a copy</h2>
+            <p>
+                Download a printable, PDF copy of this organizational chart. This version
+                is expanded to show all offices.
+            </p>
+            {# TODO Change to PDF download URL when it's available. #}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2532/commits/861674d42470d1ebe15c3ac204e66b4fa58f741c.